go.mod

```diff
@@ -8,7 +8,7 @@ require (
 	github.com/fsnotify/fsnotify v1.7.0
 	github.com/gorilla/mux v1.8.0
 	github.com/miekg/dns v1.1.35
-	github.com/opencontainers/runc v1.1.10
+	github.com/opencontainers/runc v1.1.12
```

Comment:
note that this has no effect because of a `replace` rule later in the file:

```
github.com/opencontainers/runc => github.com/opencontainers/runc v1.0.0-rc95
```

Removing that to allow the dep to be updated results in a very large change, which we don't want in openshift-sdn unless absolutely necessary.

In this case, sdn isn't using the functions that the CVE covers anyway, so there's no need to update it.